dli_app/templates/admin/edit_users.html

```diff
@@ -22,7 +22,21 @@
             <tr>
               <td>{{ user.name }}</td>
               <td>{{ user.email }}</td>
-              <td>{{ 'Yes' if user.is_admin else 'No' }}</td>
+              {% if user.email == current_user.email %}
+                  <td>Yes</td>
+              {% else %}
+                {% if user.is_admin %}
+                    <td><form action="{{ url_for('admin.demote_user', user_id=user.id) }}" method="POST">
+                      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+                      <button class="btn btn-xs btn-danger">Remove Admin</button>
+                    </form></td>
+                {% else %}
+                    <td><form action="{{ url_for('admin.promote_user', user_id=user.id) }}" method="POST">
+                      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+                      <button class="btn btn-xs btn-warning">Make Admin</button>
+                    </form></td>
+                {% endif %}
+              {% endif %}
```

Comment:
I really like this, but the "yes" seems kinda bleh compared to all the nice buttons for everyone else.
Suggestion:
````html
{% if user.is_admin %}
  <td>...etc...
    <button class="btn btn-xs btn-danger" {% if user.id == current_user.id %} disabled {% endif %}>...etc...
  </td>
{% endif %}
````

We can actually disable the button from the HTML side of things.